backend/code_review_backend/issues/migrations/0005_rename_check_issue_analyzer_check.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 4.0.5 on 2022-06-30 12:13
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("issues", "0004_issue_in_patch"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="issue",
+            old_name="check",
+            new_name="analyzer_check",
```

Comment:
Why is this change needed ? Is it required by the upgrade ?

Reply:
Yes it is, there is a function with the same name in the `django.db.models.Model` class, see https://github.com/django/django/blob/main/django/db/models/base.py#L1524. An error is raised by `runserver` if this change isn't applied:
```
django.core.management.base.SystemCheckError: SystemCheckError: System check identified some issues:

ERRORS:
issues.Issue: (models.E020) The 'Issue.check()' class method is currently overridden by <django.db.models.query_utils.DeferredAttribute object at 0x7f9e985b9eb0>.
```

Reply:
Okay gotcha, please mention it in the description :+1: